aemedge/styles/styles.css

```diff
@@ -297,8 +322,8 @@ main .button-container {
   }
 }
 
-main .background-dark .button-container {
-  .button {
+main .section.background-dark .button-container {
+  :is(.button) {
```

Comment:
Why we need `:is()` here?

Reply:
The linter was complaining about descending specificity, this stopped it